network/wsPeer.go

```diff
@@ -167,7 +167,7 @@ type wsPeerCore struct {
 	readBuffer    chan<- IncomingMessage
 	rootURL       string
 	originAddress string // incoming connection remote host
-	client        http.Client
+	client        *http.Client
```

Comment:
`GetHTTPClient` returns address of this variable, and it cannot be nil. But some client code checked it for nil